src/test/resources/pages/instructorStudentRecordsPageWithPrivateFeedback.html

```diff
@@ -128,7 +128,7 @@ <h4 class="modal-title">
     <div class="row">
       <div class="col-md-12">
         <br>
-        <div class="student_feedback panel panel-default" data-courseid="ISR.CS1101" data-fsname="Instructor feedback session" data-googleid="ISR.teammates.test" data-studentemail="teammates.test@gmail.tmt" id="studentFeedback-0">
+        <div class="student_feedback panel panel-default load-feedback-session" data-courseid="ISR.CS1101" data-fsname="Instructor feedback session" data-googleid="ISR.teammates.test" data-studentemail="teammates.test@gmail.tmt" id="studentFeedback-0">
```

Comment:
Might be due to recent changes to the JavaScript here: https://github.com/TEAMMATES/teammates/commit/cf7055ddad9c4414cd844844567fe39e0961783a#diff-0a81914fff38a10025f44b835e853fe2

Did you disable the cache in Firefox?